examples/mobile/ScheduleApp/css/style.css

```diff
@@ -0,0 +1,24 @@
+html,
+body {
+    width: 100%;
+    height: 100%;
+    margin: 0 auto;
+    padding: 0;
+    background-color: #222222;
+    color: #ffffff;
+}
+.page {
```

Comment:
These styles seems not related to the app. I don't see `page` class, `content-text` id, `contents` class in index.hrml

Reply:
Ok, this style file was generated by Tizen Studio, I will remove it.

Reply:
I removed it